Client/Frontend/Home/TopSitesPanel.swift

```diff
@@ -121,7 +121,7 @@ class TopSitesPanel: UIViewController {
         if let data = result.successValue {
             self.dataSource.setHistorySites(data.asArray())
             self.dataSource.profile = self.profile
-            self.dataSource.filterSuggestedSites()
+            self.dataSource.filterAndMergeSuggestedSites()
```

Comment:
Nit: I'd just name this `mergeSuggestedSites`. I think removing suggested sites that the user has removed would implicitly be part of that.